src/py/flwr/server/server.py

```diff
@@ -200,8 +231,14 @@ def fit_round(
         self,
         server_round: int,
         timeout: Optional[float],
+        executor: Optional[concurrent.futures.ThreadPoolExecutor],
+        pending_fs: Optional[Dict[concurrent.futures.Future, str]],  # type: ignore
     ) -> Optional[
-        Tuple[Optional[Parameters], Dict[str, Scalar], FitResultsAndFailures]
+        Tuple[
+            Optional[Parameters],
+            Dict[str, Scalar],
+            FitResultsAndFailures,
+        ]
```

Comment:
This is just formatting, right? Let's move it to a new PR that only contains trivial changes to reduce the diff in this PR